core/src/main/scala/io/qbeast/core/transform/LinearTransformation.scala

```diff
@@ -139,7 +167,17 @@ class LinearTransformationSerializer
 
 object LinearTransformation {
 
-  private def generateRandomNumber(min: Any, max: Any, seed: Option[Long]): Any = {
+  /**
+   * Creates a LinearTransformation that has random value for the nulls
+   * within the [minNumber, maxNumber] range
+   * @param minNumber
+   * @param maxNumber
+   * @param orderedDataType
+   * @param seed
+   * @return
+   */
+
+  def generateRandomNumber(min: Any, max: Any, seed: Option[Long]): Any = {
```

Comment:
probably one of these two functions needs to get deleted, right?

Reply:
Or maybe it's best to put this method on an object `TransformationUtils`, so we can use it in different parts of the code and does not affect codecov

Reply:
Good idea, I have created an object `LinearTransformationUtils` with the said function